static/app/components/groupPreviewTooltip/evidencePreview.spec.tsx

```diff
@@ -29,9 +29,10 @@ describe('EvidencePreview', () => {
   });
 
   it('shows error when request fails', async () => {
-    const api = new MockApiClient();
-    jest.spyOn(useApi, 'default').mockReturnValue(api);
-    jest.spyOn(api, 'requestPromise').mockRejectedValue(new Error());
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/issues/group-id/events/recommended/',
+      statusCode: 500,
+    });
```

Comment:
this doesn’t work anymore, because it depended on the fact that `useApiQuery` was built with `useApi`. Spying on implementation details like that isn’t great, so mocking the response is likely better.